pysatNASA/instruments/dmsp_ssusi.py

```diff
@@ -52,6 +62,7 @@
 import datetime as dt
 import functools
 
+import pysat
```

Comment:
```suggestion
from pysat import logger
```
(?) I remember there was some kind of weirdness with importing and the logger in particular. At this point though I don't remember what it is :( Something that came up during testing previously. I do remember it occurred with clearing a pysat import and reimporting the package though. I acknowledge it makes much of this comment less than perfectly helpful.Hopefully by writing it down it will free the memory before final merge.

Reply:
Upon further review, I now actually think the from pysat import logger is the problem. I'm keeping the back and forth on my thoughts as documentation for any future browsers that writing things down is essential.

Plus logging has been moved from some instrument files into general files. A heads up to those that may now about the existence of from pysat import logger in other areas.